os/fs/driver/mtd/smart.c

```diff
@@ -4408,26 +4408,6 @@ static int smart_readsector(FAR struct smart_struct_s *dev, unsigned long arg)
 		fdbg("Error reading phys sector %d\n", physsector);
 		return -EIO;
 	}
-#if SMART_STATUS_VERSION == 1
```

Comment:
Can you please specify the improvement percentage in reading without the CRC check in the commit details ?